app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,83 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre><code># apt-get install git</code></pre>
```

Comment:
what's the reason for this <pre> tags? The result seems odd with this: https://git-scm-pr-1495.herokuapp.com/download/linux

Reply:
Without the ```<pre>``` tags the code blocks will wrap all over the place and go off the screen. Here is a comment I gave above that demonstrates what happens:

> I fixed the header type for Mageia. Also, is there any way to fix the problem where the code blocks are auto-wrapping and going off the screen:
> ![Screenshot from 2020-08-10 12-46-26](https://user-images.githubusercontent.com/53054099/89824499-0e0a5180-db08-11ea-91d1-e07d353bd7de.jpg)

As you can see, now it doesn't go off the screen but it appears with a nice horizontal bar.

Reply:
We could use an alternative if you have one.

Reply:
Oh i see! :)

if we keep the `<pre>` and remove the `<code>` things get cleaner. I tried manually (compare first 2 without `code` with the last which keeps the `code`):

<img width="768" alt="imagem" src="https://user-images.githubusercontent.com/1999050/89949229-7f88f380-dc1f-11ea-9232-4249e4aa3cc0.png">

Reply:
Ok sure, I will make a commit doing that!

Reply:
Yup, you are right. It looks much better without the line around the code-block.